api/Cargo.toml

```diff
@@ -48,7 +49,7 @@ aptos-mempool = { path = "../mempool", features = ["fuzzing"] }
 aptos-secure-storage = { path = "../secure/storage" }
 aptos-temppath = { path = "../crates/aptos-temppath" }
 aptos-genesis-tool = {path = "../config/management/genesis", features = ["testing"] }
-diem-framework-releases = { path = "../aptos-move/framework/DPN/releases" }
+aptos-framework-releases = { path = "../aptos-move/framework/aptos-framework/releases" }
 aptos-sdk = { path = "../sdk" }
```

Comment:
maybe at one point this was alpha order, but you can return this back from chaos!